requirements/prod-requirements.txt

```diff
@@ -109,6 +110,7 @@ mock==2.0.0
 msgpack-python==0.5.6
 ndg-httpsclient==0.5.1
 newrelic==4.0.0.99
+nose==1.3.7
```

Comment:
shouldn't be in prod requirements, I see it comes from `leven`. Maybe you can PR a change to their `setup.py` file to move `nose` to `test_requires`.